File: Baekjoon_Online_Judge/NO_02739/2739.c
#include <stdio.h>
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2020                          | 
 | Month 09                           | 
 | Day 01                             | 
 | 구구단                             | 
 -------------------------------------- 
*/

int main(){
    int N;

    scanf("%d", &N);

    for (int i = 1; i <= 9; i ++) printf("%d * %d = %d\n", N, i, N * i);
    return 0;
}

File: Baekjoon_Online_Judge/NO_02739/2739.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# -------------------------------------- #
# | Created by happiness96             | #
# | Year 2020                          | #
# | Month 09                           | #
# | Day 01                             | #
# | 구구단                             | #
# -------------------------------------- #


if __name__ == "__main__":
    N = int(r_input())

    for i in range(1, 10):
        print(N, '*', i, '=', N * i)
